atom/browser/api/atom_api_system_preferences.cc

```diff
@@ -93,6 +93,8 @@ void SystemPreferences::BuildPrototype(
       .SetMethod("setAppLevelAppearance",
                  &SystemPreferences::SetAppLevelAppearance)
       .SetMethod("getSystemColor", &SystemPreferences::GetSystemColor)
+      .SetMethod("isTouchIDAvailable", &SystemPreferences::IsTouchIDAvailable)
+      .SetMethod("promptTouchID", &SystemPreferences::PromptTouchID)
```

Comment:
Is SystemPreferences the right place for this functionality? It doesn't seem preference-related.

Reply:
i thought about this too, but wasn't sure there was any more appropriate place 🤔